### DISCUSSION TITLE
How can I visualize the training process using tensorboard in your code?

### DISCUSSION THREAD

**Discussion by automatic0313** [open-mmlab/mmediting#1187] [How-to] [answered]
rt

**Comment by innerlee** [comment, marked as answer]
Use TensorboardLoggerHook in configs. See https://mmediting.readthedocs.io/en/latest/config.html

**Reply by likakakaka** [reply]
TensorboardLoggerHook has changed in new version. How can I visualize the training process using tensorboard in the new version?

**Reply by Z-Fran** [reply]
Please use TensorboardVisBackend following https://mmengine.readthedocs.io/en/latest/common_usage/visualize_training_log.html and https://mmagic.readthedocs.io/en/latest/user_guides/visualization.html . @likakakaka
